### DISCUSSION TITLE
Applying snapshot no fields

### DISCUSSION THREAD

**Discussion by veryphatic** [directus/directus#14667] [Archive]
Hi there, I'm trying to apply a snapshot from my local environment into my self-hosted instance. I'm using Directus 9.14.5; and the self-hosted instance is a fresh install. When I apply the schema I can see everything looks good; the collections, fields and relations appear to all be created. I can also see in the hosted postgres instance contains the fields; but when I open the Data Model in /admin I can see the collections, but there are no fields visible.

Any suggestions on what's going on?

**Comment by rijkvanzanten** [comment]
Mind restarting the directus process to rule out any caches causing the problem?

**Reply by veryphatic** [reply]
Yup, tried that. Weirdly I can apply the schema to a new local build without any issues. Btw, my cloud instance is on Digital Ocean

**Reply by rijkvanzanten** [reply]
This might sound obvious, but worth checking anyways, are you sure your copy of Directus is connected to the same database you're importing the snapshot on? 🤔
